src/main/kotlin/pl/edu/agh/gem/external/persistence/job/ExchangeRateJobEntity.kt

```diff
@@ -0,0 +1,45 @@
+package pl.edu.agh.gem.external.persistence.job
+
+import org.springframework.data.mongodb.core.mapping.Document
+import pl.edu.agh.gem.internal.job.ExchangeRateJobState
+import pl.edu.agh.gem.internal.model.ExchangeRateJob
+import java.math.BigDecimal
+import java.time.Instant
+
+@Document("Jobs")
```

Comment:
in other microservices we use  kebab case so maybe "jobs"

Reply:
changed